CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+<!-- Copyright Contributors to the Qiskit project. -->
+
+# Code of Conduct
+All members of this project agree to adhere to the Qiskit Code of Conduct listed at [https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md](https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md)
+
+----
+
+License: [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/),
+Copyright Contributors to Qiskit.
```

Comment:
Idk why we have this? The repo is Apache license 2. More generally, I don't know why you'd have a license in Code of Conduct.

cc @1ucian0

Reply:
It is not uncommon to have a license for the CoC (for exampe https://source.chromium.org/chromium/chromium/src/+/main:CODE_OF_CONDUCT.md).

With respect to the license, it is not a problem to have a file with a different license that is more permissive than the one in the repo (like in this case). Usual IANAL applies.